docs/Internationalization.md

```diff
@@ -0,0 +1,18 @@
+## Internationalization - Adding Languages
+
+Follow the steps given below
+
+### Step 1
+
+Add preferred language to application as follows
+
+![screen shot 2017-08-17 at 9 30 54 pm](https://user-images.githubusercontent.com/16130948/29444416-0b8868b2-83fd-11e7-91f0-330c11dced40.png)
```

Comment:
If you can mention how and where we can get this window  It would be really nice for other users.